tutorials/ix_gac_parmest/breakthrough_curve_parmest_demo.ipynb

```diff
@@ -0,0 +1,1072 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c8e94fa8",
+   "metadata": {},
+   "source": [
+    "# Parameter Estimation for Ion Exchange and GAC Breakthrough Data\n",
+    "\n",
+    "This demonstration will present the steps for setting up and conducting a parameter estimation for the WaterTAP ion exchange (IX) and granular activated carbon (GAC) models. \n",
+    "\n",
+    "Both technologies are potential candidates for treated water impacted by per- and polyfluoroalkyl substances (PFAS). Critical to evaluating sorptive technologies for this purpose is accurate modeling of when the effluent concentration will hit a user (or regulation) defined limit, the \"breakthrough concentration\". The breakthrough time will dictate when the media is spent and either is regenerated or, like the case with PFAS, disposed, and will have significant implications for the cost of each process. In this demonstration, we will use publicly available data from an Orange County Water District (OCWD) report to obtain calibrated kinetic, mass transfer, and isotherm parameters.The data includes the time of breakthrough in bed volumes (BV) and the effluent concentration. We want to be able to have the model accurately predict the breakthrough time for a specific species and breakthrough concentration.\n",
+    "\n",
+    "For this demonstration, we will use the Clark model for IX and the Constant Pattern Homogenous Surface Diffusion Model (CPHSDM) for GAC. Each model contains different parameters relevant to prediction of breakthrough time and is fully documented in the WaterTAP documentation.\n",
+    "\n",
+    "<p align=\"center\">\n",
+    "    <img src=\"images/report-cover.png\" alt=\"Breakthrough Curve\" width=\"400\"/>\n",
+    "</p>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "319c6ba1",
+   "metadata": {},
+   "source": [
+    "Figure 11 of the OCWD report presents the breakthrough data used in the pilot studies. In total, they evaluated eight different GAC media and four different IX resins.\n",
+    "\n",
+    "Due to the lack of breakthrough for PFOS and PFHxS, we only consider the PFOA and PFBS data for the purposes of this demonstration. Of these four species, PFOA is the only one with a limit (10 ng/L).\n",
+    "\n",
+    "<p align=\"center\">\n",
+    "    <img src=\"images/breakthrough-curves.png\" alt=\"Breakthrough Curve\" width=\"800\"/>\n",
+    "</p>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cb101f63",
+   "metadata": {},
+   "source": [
+    "### Step 0: Imports"
```

Comment:
Right before this, I would give a quick overview of all the major steps, numbered.